catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/features/CamelFeatures.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.content.monitor.features;
+
+import static org.ops4j.pax.exam.CoreOptions.maven;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.features;
+
+import java.util.Arrays;
+
+import org.ops4j.pax.exam.Option;
+
+public class CamelFeatures {
+    public enum CamelFeature {
+        XML_SPECS_API("xml-specs-api"), CAMEL("camel"), CAMEL_CORE("camel-core"), CAMEL_CATALOG(
```

Comment:
Not sure I'm a fan of those big enum files either. I'm thinking that most of the time, components will only need to pull in one or two specific dependencies from existing third-party feature files and using simple feature names should be good enough. Maintaining those enums in that case seems somewhat overkill imho.

Since this code is local to this component test, i'd be ok with leaving it for now until I'm done with my own component test. Then we can compare and start re-thinking the approach and move things to a common component test module that everyone can use.

Thoughts?